composer.json

```diff
@@ -44,9 +45,9 @@
     "sort-packages": true
   },
   "require-dev": {
+    "phpunit/phpunit": "^10.0",
```

Comment:
I suggest keeping `symfony/phpunit-bridge` for now, but creating a temporary GitHub Actions workflow that will execute `composer remove symfony/phpunit-bridge` and then `composer require  phpunit/phpunit`.

So we can test Panther using the bridge (which is necessary to detect deprecations), as well as with PHPUnit 10.

We'll remove this temporary workflow when PHPUnit Bridge will support PHPUnit 10.

Reply:
@dunglas I can do this, but the only problem is that it prevents me from being able to run tests locally.  I can remove `phpunit/phpunit` and require dev `symfony/phpunit-bridge` and use `<server name="SYMFONY_PHPUNIT_VERSION" value="10.0"/>` in `phpunit.xml.dist` but I cannot run `vendor/bin/simple-phpunit` because the bridge doesn't support phpunit 10 yet:

```
PHP Fatal error:  Uncaught Error: Class "PHPUnit\TextUI\Command" not found in /home/arderyp/panther/vendor/symfony/phpunit-bridge/Legacy/CommandForV9.php:25
Stack trace:
#0 /home/arderyp/panther/vendor/bin/.phpunit/phpunit-10.0-0/vendor/composer/ClassLoader.php(571): include()
#1 /home/arderyp/panther/vendor/bin/.phpunit/phpunit-10.0-0/vendor/composer/ClassLoader.php(428): Composer\Autoload\includeFile()
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass()
#3 /home/arderyp/panther/vendor/symfony/phpunit-bridge/TextUI/Command.php(17): class_alias()
#4 /home/arderyp/panther/vendor/bin/.phpunit/phpunit-10.0-0/vendor/composer/ClassLoader.php(571): include('...')
#5 /home/arderyp/panther/vendor/bin/.phpunit/phpunit-10.0-0/vendor/composer/ClassLoader.php(428): Composer\Autoload\includeFile()
#6 /home/arderyp/panther/vendor/bin/.phpunit/phpunit-10.0-0/phpunit(22): Composer\Autoload\ClassLoader->loadClass()
#7 /home/arderyp/panther/vendor/symfony/phpunit-bridge/bin/simple-phpunit.php(441): include('...')
#8 /home/arderyp/panther/vendor/symfony/phpunit-bridge/bin/simple-phpunit(13): require('...')
#9 /home/arderyp/panther/vendor/bin/simple-phpunit(120): include('...')
#10 {main}
  thrown in /home/arderyp/panther/vendor/symfony/phpunit-bridge/Legacy/CommandForV9.php on line 25
```
see: https://github.com/symfony/symfony/issues/49069

furthermore, I haven't tested yet, but this would probably prevent me from actually being able to use Panther with PHPUnit 10.  Does what I'm saying make sense, or do you think I am misunderstanding something?

Reply:
Yo test locally with PHPUnit 10, you'll have to do the same steps as in the CI (then revert the changes in `composer.json` before committing).

This is definitely not ideal, but quite common an only temporary. We'll be able to remove this hack when PHPUnit Bridge will support PHPUnit 10.

As PHPUnit is not a hard dependency of Panther, this will not prevent users to use Panther with PHPUnit 10 in their own projects.

TL;DR: restore the content of Panther's `composer.json` and change it only in a specific CI workflow.

Reply:
Thanks for the info.

I am inexperienced with GitHub workflows. Should I be looking into changing our github workflow somehow, or are you just saying "revert back to phpunit-bridge, which will allow Github workflow to work properly, and just manually add/remove phpunit for my own testing locally"?

I am swamped tomorrow but hopefully can carve out some time in the afternoon, otherwise I can look on Friday.

Reply:
done: https://github.com/arderyp/panther/commit/cdcd7e08b58d93bf10a74c67cc6dbbde097dee89

@dunglas please resolve if sufficient